intellij-ext/gradle.properties

```diff
@@ -19,7 +19,7 @@ platformVersion = 2022.2
 platformPlugins = org.jetbrains.plugins.go:222.3345.118, org.intellij.intelliLang
 
 # Java language level used to compile sources and to generate the files for - Java 11 is required since 2020.3
-javaVersion = 11
+javaVersion = 17
```

Comment:
Why is this necessary?

Reply:
- The Java configuration specifies sourceCompatibility=11 but IntelliJ Platform 2022.2 requires sourceCompatibility=17.
See: https://jb.gg/intellij-platform-versions

when run `./gradlew buildPlugin`

Reply:
i believe this could be the reason: https://blog.jetbrains.com/platform/2022/08/intellij-project-migrates-to-java-17/
but didn't sound relevant since it says its needed for 2022.3, but here https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#intellij-platform-based-products-of-recent-ide-versions it says it is needed for 2022.2